thevetdoctor/wayfarer
# #167112681 Refactor Code

What does this PR do?
Attempts to refactor the codebase

Description of Task to be completed?
Remove redundant code and make the code more efficient

How should this be manually tested?
Clone the repo, and run 'npm start'
The code should run faster

Any background context you want to provide? NA
What are the relevant pivotal tracker stories?
#167112681

Screenshots (if appropriate) NA
Questions: NA
